LANDING_PAGE_REDESIGN.md

```
@@ -0,0 +1,286 @@
+- Review Stimulus Handbook for controller patterns
+
+---
+
+**Built with ❤️ using:**
+- Ruby on Rails 8.0
+- Stimulus JS
+- Tailwind CSS
+- Modern CSS3 Animations
+- Intersection Observer API
+
+*Last Updated: September 30, 2024*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)